contrib/metadata-types/config.yaml

```diff
@@ -37,5 +37,5 @@ quicktype_config:
   # rust: ~
   schema: ~
   # swift: ~
-  # typescript:
+  typescript:
```

Comment:
This also requires running `yarn generate-types` / committing the new generated TypeScript module.